egs2/README.md

```diff
@@ -146,6 +146,7 @@ See: https://espnet.github.io/espnet/espnet2_tutorial.html#recipes-using-espnet2
 | reverb                  | REVERB (REverberant Voice Enhancement and Recognition Benchmark) challenge                                                       | ASR                     | ENG                  | https://reverb2014.dereverberation.com/                                                                      |              |
 | ru_open_stt             | Russian Open Speech To Text (STT/ASR) Dataset                                                                                    | ASR                     | RUS                  | https://github.com/snakers4/open_stt                                                                         |              |
 | ruslan                  | RUSLAN: Russian Spoken Language Corpus For Speech Synthesis                                                                      | TTS                     | RUS                  | https://ruslan-corpus.github.io/                                                                             |              |
+| sdsv_2021                   | SdSV 2021: Short-duration Speaker Verification (SdSV) Challenge 2021                                                        | SPK                     | 10+ Languages           | https://sdsvc.github.io/                                                                     |              |
```

Comment:
```suggestion
| sdsv21                   | SdSV 2021: Short-duration Speaker Verification (SdSV) Challenge 2021                                                        | SPK                     | 10+ Languages           | https://sdsvc.github.io/                                                                     |              |
```
Please follow the naming convention with the other recipes.

Reply:
Noted. I'll update the task name as well.

Reply:
I have updated the README as well as task name to egs2/sdsv21